addons/hr_attendance/controllers/main.py

```diff
@@ -138,11 +138,15 @@ def scan_barcode(self, token, barcode):
 
     @http.route('/hr_attendance/manual_selection', type="json", auth="public")
     def manual_selection(self, token, employee_id, pin_code):
+        return self.manual_selection_with_geolocation(token, employee_id, pin_code)
+
+    @http.route('/hr_attendance/manual_selection_with_geolocation', type="json", auth="public")
+    def manual_selection_with_geolocation(self, token, employee_id, pin_code, latitude=False, longitude=False):
```

Comment:
it's good to duplicate the method for stable I think (so people overriding the original ones), for the route I don't know if it's necessary or if we could just keep the route "/hr_attendance/manual_selection" on the new method